vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>Why is this cell being shown here again like above, and in a section for "Generate with Imagen"? It seems a bit out of place</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>This is for uploading the mask, the previous image is for the base image. I've gone ahead and separated the sections in two for clarity. </p>